Add files via upload

VladimirtheGreatest opened the pull request:
Masterpiece6 + additonal changes and queries for mobile navigation.

VladimirtheGreatest committed:
Add files via upload

Masterpiece6 + additonal changes and queries for mobile navigation.

stephairth commented on css/style.css:
```diff
@@ -0,0 +1,2451 @@
+    max-width: 180px;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+    right: 210px;
```
change "right: 210px" to "right: 160px" to re-align the language dropdown, please!

VladimirtheGreatest replied:
changed

stephairth reviewed (commented):
Please change "right: 210px" at line 1743 to "right: 160px"--otherwise looks good! Is there any way we can make the images in main section 2 less stretched?

stephairth committed:
add button aria-labels and link names for a11y

sasigit7 committed:
Added two compressed images

Converted two images battlepass.png and mobile.png to battlepass.webp and mobile.webp for faster downloads and less data consumption.

stephairth reviewed (approved):
compressed images look good to me

sasigit7 committed:
Update Image extensions

Updated (battlepass (Line: 1264)& mobile(Line: 58)) image extensions from png to webp formats in style.css

stephairth committed:
make language dropdown accessible by keyboard

stephairth committed:
remove nav dropdown keyboard access til working

sasigit7 commented:
> Please change "right: 210px" at line 1743 to "right: 160px"--otherwise looks good! Is there any way we can make the images in main section 2 less stretched?

sasigit7 commented:
Pheww!! Apologies!! By mistake, I closed the pull request. Reopening it.

VladimirtheGreatest committed:
Update style.css
